### ISSUE TITLE
CLI didn't catch type errors 

### ISSUE THREAD

**Issue by Krakof** [allegro/typescript-strict-plugin#72]
Hi,
Thanks a lot for the package, it helps us to add strict mode to the project based on lerna monorepo module by module.

Problem:
During react upgrade to v18 we upgraded next packages as well:
```
    "@testing-library/dom": "9.3.1" -> "9.3.3",
    "@testing-library/jest-dom": "5.16.5" -> "6.2.0",
    "@testing-library/react": "11.2.7" ->  "14.1.2",
    "@testing-library/user-event": "14.4.3" -> "14.5.2"
```
After that, script `"typecheck": "tsc-strict --noEmit --incremental false"` runs successfully, despite type errors (IDE still shows errors).
I found out that update of the listed packages to specific versions:
```
`@testing-library/jest-dom: 6.x.x`
"@testing-library/user-event": 14.5.x
```
breaks cli check.
We are using package-lock file in project.
Re-generating package-lock breaks cli as well, but there are too much changes to define a problem.

Any thought will be helpful. Thanks in advance

**Comment by alanmoo** [comment]
I'm experiencing something similar; I wonder if `tsc-strict` is only checking for errors that violate strictness, but somehow not checking for other typescript errors. The workaround here is to (admittedly as the docs indicate) run typechecks using `tsc && tsc-strict`
